README.md

```diff
@@ -52,7 +52,7 @@ php app/console  doctrine:migrations:migrate
 in the vagrant machine (in directory `/vagrant` ) run: (in case you want to use the grant type `password`)
 
 ```
-php app/console acme:oauth-server:client:create --grant-type="password" --grant-type="refresh_token" --grant-type="token"
+php app/console acme:oauth-server:client:create --grant-type="password" --grant-type="refresh_token" --grant-type="token" --client-type="backend"
```

Comment:
also explain in a note section , why we need this --client-type,  i.e that we can't use the "scope" as defined by standard oauth2 , because the bundle we're using does not implement scope at the client level , (with a link to the two issue on the bundle)